client/scss/components/_header.scss

```diff
@@ -149,19 +119,12 @@
   }
 }
 
-// Media for Windows High Contrast
-@media (forced-colors: $media-forced-colours) {
-  .w-header .field-content {
-    border: 0.1em solid $system-color-link-text;
-  }
-}
-
 @include media-breakpoint-up(sm) {
   .w-header {
     .row {
       padding-top: 1.5rem;
-      padding-inline-start: 90px;
-      padding-inline-end: 90px;
+      padding-inline-start: $desktop-nice-padding;
+      padding-inline-end: $desktop-nice-padding;
```

Comment:
double check this - when I set `$desktop-nice-padding` it was using the 50px not 90px value (looks like the variable is set twice)

Reply:
I can only see it set once to 80px so I believe we’re good 🤔 Your issue might have been with this other declaration on `main`? https://github.com/wagtail/wagtail/blob/405af5d4515d44c846e1134aa355352a37ff3377/client/scss/layouts/_login.scss#L2

Reply:
Yep. That was it.